docs/docs/reference/cli.md

```diff
@@ -17,9 +17,9 @@ Commands:
   * [instance-types](#constellation-config-instance-types): Print the supported instance types for all cloud providers
 * [create](#constellation-create): Create instances on a cloud platform for your Constellation cluster
 * [init](#constellation-init): Initialize the Constellation cluster
-* [mini](#constellation-mini): Manage MiniConstellation clusters
-  * [up](#constellation-mini-up): Create and initialize a new MiniConstellation cluster
-  * [down](#constellation-mini-down): Destroy a MiniConstellation cluster
+* [mini](#constellation-mini): Manage mini Constellation clusters
+  * [up](#constellation-mini-up): Create and initialize a new mini Constellation cluster
+  * [down](#constellation-mini-down): Destroy a mini Constellation cluster
```

Comment:
@m1ghtym0 is this wanted?

Reply:
No, I'm confused. I thought I've updated the source of these strings in the code, which seems not to be the case.

Reply:
Which of the two versions did you expect? `mini constellation`?

Reply:
I expected `MiniConstellation`

Reply:
Did you change the generated file stead of the description in code?

Reply:
Yes, these were changes in an auto-generated file https://github.com/edgelesssys/constellation/commit/c7010f76ca64f3d4db80b3c3730fe6aa6ba84627#diff-350d4ed56af88d542f09d4ef8d98800e57aefe3d48ecfa18543c8a9360c14e35

Reply:
Thanks!